nilearn/plotting/find_cuts.py

```diff
@@ -209,6 +210,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
 
     data = _smooth_array(data, affine, fwhm='fast')
 
+    if (isinstance(n_cuts, numbers.Number) and n_cuts > 0):
```

Comment:
I think we should stop using magical values such as 1e-10 everywhere in the code. We should create a function in utils once and for all.

Removing abs is out of question: we have to be robust to the direction of the error.

Also, 1e-10 may not work for float32:
```
In [2]: np.finfo(np.float32).eps
Out[2]: 1.1920929e-07
```